server/server.js

```diff
@@ -99,729 +54,16 @@ app.use(
-        positionTitle: participantInfo.positionTitle,
-        positionType: participantInfo.positionType,
-        hiredDate: participantInfo.hiredDate,
-        startDate: participantInfo.startDate,
-      });
-
-      logger.info({
-        action: 'hired_participant_post',
-        performed_by: {
-          username: user.username,
-          id: user.id,
-        },
-        participant_id: response.id,
-      });
-
-      return res.status(201).json(response);
-    } catch (excp) {
-      return res.status(400).send(`${excp}`);
-    }
-  })
-);
-
-// Create participant records from uploaded XLSX file
-app.post(
-  `${apiBaseUrl}/participants/batch`,
-  keycloak.allowRolesMiddleware('maximus'),
-  keycloak.getUserInfoMiddleware(),
-  multer({
-    fileFilter: (req, file, cb) => {
-      if (file.fieldname !== 'file') {
-        req.fileError = 'Invalid field name.';
-        return cb(null, false);
-      }
-      if (file.mimetype === 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet') {
-        return cb(null, true);
-      }
-      req.fileError = 'File type not allowed.';
-      return cb(null, false);
-    },
-  }).single('file'),
-  asyncMiddleware(async (req, res) => {
-    if (req.fileError) {
-      return res.json({ status: 'Error', message: req.fileError });
-    }
-
-    try {
-      const response = await parseAndSaveParticipants(req.file.buffer);
-      const user = req.hcapUserInfo;
-      logger.info({
-        action: 'participant_post',
-        performed_by: {
-          username: user.username,
-          id: user.id,
-        },
-        // Slicing to one page of results
-        ids_posted: response.slice(0, 10).map((entry) => entry.id),
-      });
-
-      return res.status(201).json(response);
-    } catch (excp) {
-      return res.status(400).send(`${excp}`);
-    }
-  })
-);
-
-// Get linked participants for users
-app.get(
-  `${apiBaseUrl}/user/participants`,
-  keycloak.setupUserMiddleware(),
-  asyncMiddleware(async (req, res) => {
-    const { email, user_id: userId } = req.user;
-    if (email && userId) {
-      const response = await getParticipantsForUser(userId, email);
-      logger.info({
-        action: 'user_participant_get',
-        performed_by: userId,
-        id: response.length > 0 ? response[0].id : '',
-      });
-      res.status(200).json(response);
-    } else {
-      res.status(401).send('Unauthorized user');
-    }
-  })
-);
-
-app.post(
-  `${apiBaseUrl}/participants`,
-  asyncMiddleware(async (req, res) => {
-    await validate(ParticipantSchema, req.body);
-
-    const participant = {
-      ...req.body,
-      formVersion: 'v2',
-      maximusId: null,
-      postalCodeFsa: req.body.postalCode.substr(0, 3),
-      preferredLocation: req.body.preferredLocation.join(';'),
-      interested: 'yes',
-      nonHCAP: null,
-      crcClear: null,
-      callbackStatus: false,
-      userUpdatedAt: new Date().toJSON(),
-    };
+app.use(`${apiBaseUrl}`, apiRouter);
```

Comment:
Is the psiRouter going to need to be placed in the apiRouter once this feature is merged?